app/models/candidate.rb

```diff
@@ -1,3 +1,5 @@
 class Candidate < ActiveRecord::Base
   validates :name, presence: true
+  validates_format_of :photo_url, with: URI.regexp
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.